tests/test_graphical_units/test_img_and_svg.py

```diff
@@ -48,8 +41,8 @@ class RhomboidTest(Scene):
     """Test the default fill and parsed stroke of a rhomboid"""
 
     def construct(self):
-        rhomboid = SVGMobject(get_test_resource("rhomboid.svg")).scale(0.5)
-        rhomboid_fill = rhomboid.copy().set_fill(opacity=1).shift(UP * 2)
+        rhomboid = SVGMobject(get_svg_resource("rhomboid.svg")).scale(0.5)
+        rhomboid_fill = rhomboid.copy().set_fill(color=WHITE, opacity=1).shift(UP * 2)
```

Comment:
This is the one line I had trouble making sense of - how can this line specify the fill color as white, but the test file not need a change? Are the results as you expect them to be? I think the fill should have been black as the default SVG color, and that wouldn't have shown up on the black background that is default for Manim...

Reply:
Yea that was left in by mistake thanks for spotting it, I removed it because it is already white in the svg